src/battle-log.ts

```diff
@@ -843,9 +843,15 @@ class BattleLog {
 				const videoId = /(?:\?v=|\/embed\/)([A-Za-z0-9_\-]+)/.exec(src)?.[1];
 				if (!videoId) return {tagName: 'img', attribs: ['alt', `invalid src for <youtube>`]};
 
+				const time = getAttrib('start') || /(?:\?|&)t=([0-9]+)/.exec(src)?.[1] || false;
+
 				return {
 					tagName: 'iframe',
-					attribs: ['width', width, 'height', height, 'src', `https://www.youtube.com/embed/${videoId}`, 'frameborder', '0', 'allow', 'accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture', 'allowfullscreen', 'allowfullscreen'],
+					attribs: [
+						'width', width, 'height', height,
+						'src', `https://www.youtube.com/embed/${videoId}${time ? `?start=${time}` : ''}`,
```

Comment:
Does `t=` not work in embeds?

Reply:
What do you mean? `url?t=` unto itself, with nothing else after it?

Reply:
No, I mean, you only accept `t=` in the URL, and you only use `start=` in the embed URL. Should we all be searching for `start=` in the input URL?

Reply:
Also, are attributes sanitized in this? What happens if I try to use an attribute of `start="<i>test"`?